src/pkg/services/m365/api/lists.go

```diff
@@ -53,6 +53,7 @@ const (
 	// tracks to backup/restore supportability of `documentLibrary` templated lists
 	DocumentLibraryListTemplateName = "documentLibrary"
 	SharingLinksListTemplateName    = "sharingLinks"
+	AccessRequestsListTemplateName  = "accessRequest"
```

Comment:
minor nit: `Name` suffix is redundant for the templates and can be removed. I do see we are using the same convention for all var namings in this file. Usually we avoid such suffixes as it can make code verbose. 

Also, I don't think we need to export any of these vars, since their usage is restricted to api pkg.

Reply:
Using `WebTemplateExtensionsListTemplateName` and `DocumentLibraryListTemplateName` in `collection/site/restore_test.go`, hence exported. Others are exported for parity. I will remove the `Name` suffix.

Reply:
Ah thanks, I didn't notice that we were using a template in restore tests.